### ISSUE TITLE
'dotnet_style_require_accessibility_modifiers = always' doesn't work for interface members

### ISSUE THREAD

**Issue by TessenR** [dotnet/roslyn#74244]
**Version Used**: 
```
Microsoft Visual Studio Professional 2022
Version 17.10.0 Preview 6.0
VisualStudio.17.Preview/17.10.0-pre.6.0+34825.169
Microsoft .NET Framework
Version 4.8.09037
```

**Steps to Reproduce**:

Set the option in editorconfig:
```
dotnet_style_require_accessibility_modifiers = always:error
```

Compile the following code:
```cs
public interface IInterface
{
  void Method();
}
```

**Diagnostic Id**:

`IDE0040 Add accessibility modifiers`

**Expected Behavior**:
There should be an error indicating that a `public` modifier for `void Method()` is required

**Actual Behavior**:
There's no IDE0040 inspection. `dotnet_style_require_accessibility_modifiers = always` is behaving the same as `dotnet_style_require_accessibility_modifiers = for_non_interface_members`

**Notes**:
This code looks outdated, there's already a version of C# that supports modifiers on interface members:
https://github.com/dotnet/roslyn/blob/f61885262640ef325336468ae2368e26217471da/src/Analyzers/CSharp/Analyzers/AddAccessibilityModifiers/CSharpAddAccessibilityModifiersDiagnosticAnalyzer.cs#L56-L65

**Comment by CyrusNajmabadi** [comment]
We would take a targeted community PR here.
